### PR TITLE
RHDEVDOCS-3859 Review RNs for OCP 4.9.z bug fix update

### PR DESCRIPTION
**Pull request by rishumehra**
* Aligned team: Dev Tools
* For branches: Enterprise 4.9.z release notes
* Jira: [RHDEVDOCS-3859](https://issues.redhat.com/browse/RHDEVDOCS-3859)
* Direct link to doc preview: https://deploy-preview-43463--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes
* SME review: Received in the individual BZs.
* QE review: @sanketpathak
* Peer review: @max-cx @opayne1

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -1729,6 +1729,8 @@ The following update for the Performance Addon Operator is now available for {pr
 
 * Previously, the log file contained information in a single line without any line breaks. In the current release, the log file contains the expected line breaks with additional line breaks around log headers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1985080[*BZ#1985080*])
 
+* Previously, the browser used to crash or display a white blank page when you open the *Edit Deployment* page for a non-existing Deployment resource. In the current release, a 404 error page is rendered instead of a browser crash or a white blank page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002273[*BZ#2002273*])
```
Comment (max-cx) [inline release_notes/ocp-4-9-release-notes.adoc:1732]:
```suggestion
* Previously, opening the *Edit Deployment* page for a non-existent Deployment resource resulted in a blank browser tab. In the current release, opening the *Edit Deployment* page for a non-existent Deployment resource results in a 404 error page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002273[*BZ#2002273*])
```

**Review by opayne1 — COMMENTED** [review]
Just a few things to consider.

**Review by opayne1 — COMMENTED** [review]
LGTM!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (rishumehra) [reply release_notes/ocp-4-9-release-notes.adoc]: Done

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -2659,7 +2659,7 @@ Starting with {product-title} 4.9.24, support for using the Cloud Credential Ope
 [id="ocp-4-9-24-bug-fixes"]
 ==== Bug fixes
 
-* Previously, the console failed to upload an edit page for *Deployment*, *DeploymentConfig*, and *Serverless Service*. This update adds a check for a loading error, and the page now displays a *Not Found* message rather than failing to load. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002273[*BZ#2002273*])
+* Previously, opening the *Edit Deployment* page for a non-existent Deployment resource resulted in a blank browser tab. In the current release, opening the *Edit Deployment* page for a non-existent Deployment resource results in a 404 error page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002273[*BZ#2002273*])
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply release_notes/ocp-4-9-release-notes.adoc]: @rishumehra please check the instructions for usage of the term 'deployment' in the OCP docs GH Documentation guidelines.

**Pull request merged**